Minio/DataModel/ObjectOperationsArgs.cs

```diff
@@ -1778,8 +1778,8 @@ internal override void Validate()
                                         " should be set.");
         if (!string.IsNullOrWhiteSpace(FileName)) utils.ValidateFile(FileName);
         // Check object size when using stream data
-        if (ObjectStreamData != null && ObjectSize == 0)
-            throw new ArgumentException($"{nameof(ObjectSize)} must be set");
+        if (ObjectStreamData != null && ObjectSize < 0)
+            throw new ArgumentException($"{nameof(ObjectSize)} cannot be less than 0.");
```

Comment:
We set `ObjectSize` to `-1` when `ObjectStreamData` is set. That' why the original error message was saying `{nameof(ObjectSize)} must be set` as `0` size means `ObjectSize` variable is not set.

Reply:
Just checked one more time. In the current implementation size=0, means size is not defined. But you can `PUT` a file of size `0` by setting its size to `-1`, which seems to be used for unknown size.
I think we don't need this check at all.

Reply:
@haiduong87 

You also need to make sure to run `regitlint` after making your changes as a last step just before `git push`.

Reply:
@ebozduman @harshavardhana 
So we don't need to check the size. Just like the 1st commit.

I will make another submit